docker-compose.yml

```diff
@@ -68,6 +68,12 @@ services:
       validator_base_path: validator
     mem_limit: 1000m
     restart: unless-stopped
+  inferno_bulk_data_server:
+    image: infernocommunity/bulk-data-server
+    mem_limit: 1500m
+    restart: unless-stopped
+    environment:
+      - SERVER_BASE_URL=http://10.14.196.25
```

Comment:
What does setting this environment variable do?  It looks a little out of place...

Will this work no matter what server it is deployed on (inferno, infernotest, localhost)?

Reply:
Unfortunately for now this needs to be hard coded to match where it is deployed so the system knows what url it is at. I am working on another ticket to fix this issue. https://oncprojectracking.healthit.gov/support/browse/FI-951

Reply:
might be worth investigating this a little more to see if we can get around this.  Otherwise we'll want clear instructions on how to deploy this.

Reply:
Yeah, we can't merge this until this works out of the box for whoever runs it (with the one exception of SSL certs, which is an annoying flag we need to switch on and off).

Reply:
Sounds good, I'll try to handle issue 951 in this (and the bulk-data-server PR https://github.com/inferno-community/bulk-data-server/pull/1) instead.

Reply:
Have updated to remove the need for this env var (though some returned urls from the fhir services may be incorrect)